### PR TITLE
Add alertmanager, ability to specify custom configmaps

### PR DESCRIPTION
**Pull request by afoninsky**

### CODE REVIEW

File: charts/victoria-metrics-agent/values.yaml [hunk 1]
```diff
@@ -76,6 +75,11 @@ affinity: {}
 
 # vmagent scraping configuration:
 # https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/docs/vmagent.md#how-to-collect-metrics-in-prometheus-format
+
+# use existing configmap if specified
+# otherwise .config values will be used
+configMap: ""
```
Comment (tenmozes) [inline charts/victoria-metrics-agent/values.yaml:81]: please create a helper which can define what name should be used

File: charts/victoria-metrics-agent/values.yaml [hunk 2]
```diff
@@ -27,7 +27,6 @@ serviceAccount:
   name:
 
 # WARN: need to specify at least one remote write url
-remoteWriteUrls: []
```
Comment (tenmozes) [inline charts/victoria-metrics-agent/values.yaml:30]: please don't delete the var

File: charts/victoria-metrics-alert/test.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+server:
```
Comment (tenmozes) [inline charts/victoria-metrics-alert/test.yaml:1]: should it be here?

File: charts/victoria-metrics-agent/values.yaml (hunk 2 above)
Comment (afoninsky) [reply charts/victoria-metrics-agent/values.yaml]: sorry, forgot to return

File: charts/victoria-metrics-agent/values.yaml (hunk 1 above)
Comment (afoninsky) [reply charts/victoria-metrics-agent/values.yaml:82]: nice trick, thanks for pointing it

**Pull request merged**